src/components/Input/Input.tsx

```diff
@@ -14,7 +14,7 @@ export type InputProps = PropsWithBox<
   > & {
     label?: ReactNode;
     error?: boolean;
-    type?: "text" | "number";
+    type?: "text" | "number" | "url" | "email" | "password";
```

Comment:
question: would be better to use build-in types `InputHTMLAttributes<HTMLInputElement>['type']`?

Reply:
We have had this before but then it allows users to pass a lot of options that we currently do not support e.g `date` or `time`. That is why I opted for adding type manually.

Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#input_types